### PR TITLE
[jira-integration-v2-sprint-2] allow popover on delete X for listed o…

### PR DESCRIPTION
**Pull request by jfer-rubio**
…ptions

Popover overlay is now available and will be shown if set && state.openPopover
- [ ] @gmjorge 
- [ ] @diegoglezs

### CODE REVIEW

**Comment by jfer-rubio** [comment]
depends on https://github.com/wizeline/wizeline-webapp/pull/2160

File: src/Select.js [hunk 1]
```diff
@@ -66,10 +66,11 @@ var Select = React.createClass({
 		valueRenderer: React.PropTypes.func,        // valueRenderer: function (option) {}
 
         /* New for wizeline react-select */
-        listReadOnlyMode: React.PropTypes.bool,	   // Non editable list mode currently implemented for List select only
+    listReadOnlyMode: React.PropTypes.bool,	   // Non editable list mode currently implemented for List select only
```
Comment (go-diego-go) [inline src/Select.js]: is the indentation right here?

File: src/Value.js [hunk 2]
```diff
@@ -57,11 +72,14 @@ var Value = React.createClass({
...
 
 		var removeIcon;
+		var popOverContent = this.state.popOverOpen ? this.props.overlay : null;
 		if (this.props.deletable) {
 			removeIcon = React.createElement(
 				'span',
 				{ className: 'Select-item-icon',
 					onMouseDown: this.blockEvent,
+					onMouseEnter: this._showPopover,
+	        onMouseLeave: this._hidePopover,
```
Comment (go-diego-go) [inline src/Value.js]: same question about indentation

**Comment by go-diego-go** [comment]
do we normally commit the `dist/react-select.min.js` file?

File: src/Value.js [hunk 3]
```diff
@@ -25,6 +26,20 @@ var Value = React.createClass({
 		}
 	},
 
+  getInitialState () {
```
Comment (bit-dragon) [inline src/Value.js]: Indentation seems to be wrong here, I think the whole document has 8 space of indentation.

**Comment by bit-dragon** [comment]
@diegoglezs The minified file should be included.

@jfer-rubio the code looks good to me but I think the indentation looks wrong here, not sure if is something with github, can you verify if is right?

**Comment by jfer-rubio** [comment]
@gmjorge  @diegoglezs , seems like this projects uses tabs instead of spaces for identation, my text editor should be changing it. I'll try to disable that configuration

File: dist/react-select.js [hunk 4]
```diff
@@ -1138,6 +1141,20 @@ var Value = React.createClass({
 		}
 	},
 
+	getInitialState: function getInitialState() {
+		return {
+			popOverOpen: false
```
Comment (juanitobebe) [inline dist/react-select.js:1146]: This was already merged and this is super minor but Bootstrap docs refer to popover as `Popover` so the casing should be: `popoverOpen` IMO.

**Pull request merged**